yarn.lock

```diff
@@ -877,7 +877,7 @@
 
 "@keep-network/ci@github:keep-network/ci":
   version "0.0.1"
-  resolved "git+ssh://git@github.com/keep-network/ci.git#48512f121e1f7ba87fa19ee0cd8924a8256774ef"
+  resolved "https://codeload.github.com/keep-network/ci/tar.gz/d61015051a3dc9679dd5cc3d6b8b4640e5d93bde"
```

Comment:
I'm not sure if this change from `git+ssh://git@github.com` to `https://codeload.github.com` is ok.